hosts/azazel/ogion.cz/bag/default.nix

```diff
@@ -33,19 +27,14 @@ let
     domain_name = "https://bag.ogion.cz";
     server_name = "Wallabag";
 
-    mailer_transport = "sendmail";
-    mailer_user = null;
-    mailer_password = null;
-    mailer_host = null;
-    mailer_port = null;
-    mailer_encryption = null;
-    mailer_auth_mode = null;
+    mailer_dsn = "native://default";
```

Comment:
Looks like Symfony Mailer does not support getting `sendmail_path` from `php.ini` like [SwiftMailer did in the end](https://github.com/symfony/swiftmailer-bundle/commit/31a4fed8f621f141ba70cb42ffb8f73184995f4c), so `sendmail://default` will fail due to non-existent `/usr/sbin/sendmail`. Opened https://github.com/symfony/symfony/pull/51749

`native://default` is [supposed to respect the config](https://symfony.com/doc/current/mailer.html#using-built-in-transports) but that makes sign in fail with:

> The "native" scheme is not supported.